src/BooleanValueSender.cpp

```diff
@@ -0,0 +1,35 @@
+#include "BooleanValueSender.h"
+
+/*
+Right now, the table is /SmartDashboard so we can visualize the values sent from the Orin.
+Later, this should be changed to a new table, such as /Orin or /Vision, to avoid populating the SmartDashboard with unnecessary values
+*/
+
+
+BooleanValueSender::BooleanValueSender(std::string key){
+  inst_ = nt::NetworkTableInstance::GetDefault();
+  inst_.SetServer("10.7.66.2");
```

Comment:
Instead of hardcoding this IP Address in each of these files, let's add a NetworkTablesConfig.h file and put a constant like this:

```
const std::string SERVER_IP_ADDRESS = "10.7.66.2"
```
and then include the NetworkTablesConfig.h in each of the header files.  This way if it ever changes we can change it in one place rather than multiple places.

Reply:
great idea. Fixed!